src/containers/Publish/index.tsx

```diff
@@ -251,6 +251,7 @@ function LinearProgressWithLabel(props: LinearProgressProps & { value: number })
           config: transformationConfig,
           source: tempRefinedCsv || parsedCsv,
           type: "rml",
+          wizardAppConfig: wizardAppConfig,
```

Comment:
I'm worried including the full config might lead to 
```suggestion
          prefixes: await wizardAppConfig.getPrefixes(),
```